addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/OnkyoBindingConstants.java

```diff
@@ -69,6 +70,7 @@
     public static final ThingTypeUID THING_TYPE_TXNR646 = new ThingTypeUID(BINDING_ID, ONKYO_TYPE_TXNR646);
     public static final ThingTypeUID THING_TYPE_TXNR656 = new ThingTypeUID(BINDING_ID, ONKYO_TYPE_TXNR656);
     public static final ThingTypeUID THING_TYPE_TXNR717 = new ThingTypeUID(BINDING_ID, ONKYO_TYPE_TXNR717);
+    public static final ThingTypeUID THING_TYPE_TXNR708 = new ThingTypeUID(BINDING_ID, ONKYO_TYPE_TXNR708);
```

Comment:
Can you move this line before the line with `THING_TYPE_TXNR717` so the list is nicely sorted?